cli.py

```diff
@@ -249,6 +254,69 @@ def add_metric(
     )
 
 
+@cli
+def load_stats_history(env: str = "demo", since: str = "2024-04-02"):
+    parsed_since = date.fromisoformat(since)
+    print(f"Loading stats history since {since}...")
+    today = date.today()
+    for d in range((today - parsed_since).days):
+        current_date = parsed_since + timedelta(d)
+        load_stats(env=env, day=current_date.isoformat())
+
+
+@cli
+def load_stats(env: str = "demo", day: str | None = None):
+    """
+    Upsert the stats table from Matomo
+    """
+    # defaults to yesterday
+    parsed_day = date.fromisoformat(day) if day else date.today() - timedelta(days=1)
+    print(f"Loading stats for {parsed_day.isoformat()}...")
+
+    stats_url = get_config_value(env, "stats_url")
+    stats_site_id = get_config_value(env, "stats_site_id")
+    stats_token = get_config_value(env, "stats_token")
+
+    if not stats_url or not stats_site_id or not stats_token:
+        print("Skipping stats loading: missing config value(s)")
```

Comment:
Might be worth a `log.warn`

Reply:
Ideally yes, but we don't have a logger yet. I'd keep this as is for now (not an error), and add a logger later.